rpm/libsystrace.spec

```diff
@@ -3,7 +3,7 @@ Version: 0.0.0
 Release: 1
 Summary: A library for logging systrace data
 License: BSD
```

Comment:
The correct SPDX identifier would be `BSD-3-Clause`

Reply:
Fixed.